examples/connext_dds/basic_security/py/secure_sub.py

```diff
@@ -0,0 +1,91 @@
+#
+# (c) 2023 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+# RTI grants Licensee a license to use, modify, compile, and create derivative
+# works of the Software solely for use with RTI products.  The Software is
+# provided "as is", with no warranty of any type, including any warranty for
+# fitness for any purpose. RTI is under no obligation to maintain or support
+# the Software.  RTI shall not be liable for any incidental or consequential
+# damages arising out of the use or inability to use the software.
+#
+
+import rti.connextdds as dds
+from hello import HelloWorld
+import time
+import argparse
+import textwrap
+import pathlib
+
+
+XML_FILE_PATH = (
+    str(pathlib.Path(__file__).parent.absolute()) + "/SecureQos.xml"
+)
+
+# Process incoming data in a listener, print out each sample
+
+
+class SecurityListener(dds.NoOpDataReaderListener):
+    def on_data_available(self, reader):
+        for sample in reader.take_data():
+            print(sample)
+
+
+def subscriber_main(domain_id, sample_count, profile):
+    print(f"Running with profile {profile}")
+    provider = dds.QosProvider(XML_FILE_PATH)
```

Comment:
Interesting, nice not to depend on the default `USER_QOS_PROFILES.xml` name.
On the other hand, all other examples use `USER_QOS_PROFILES.xml`. I'm not enforcing anything, do what you prefer.